Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -814,6 +908,17 @@ def create_channel(team_aad_id: str, channel_name: str, channel_description: str
     return channel_id
 
 
+def create_channel_command():
+    channel_name: str = demisto.args().get('channel_name', '')
+    channel_description: str = demisto.args().get('description', '')
+    team_name: str = demisto.args().get('team', '')
+    team_aad_id = get_team_aad_id(team_name)
+
+    channel_id: str = create_channel(team_aad_id, channel_name, channel_description)
+    if channel_id:
+        demisto.results(f'The channel "{channel_name}" was created successfully')
```

Comment:
Why not also add the `channel_id` to context? I think this could help our users in their next tasks